Marlin/src/lcd/extui/lib/mks_ui/draw_ui.cpp

```diff
@@ -137,12 +137,12 @@ void gCfgItems_init() {
   gCfgItems.levelingPos[3][1] = Y_MAX_POS - 30;
   gCfgItems.levelingPos[4][0] = X_BED_SIZE / 2;
   gCfgItems.levelingPos[4][1] = Y_BED_SIZE / 2;
-  gCfgItems.cloud_enable  = true;
-  #if ENABLED(MKS_WIFI_MODULE)
+  gCfgItems.cloud_enable      = false;
+  //#if ENABLED(MKS_WIFI_MODULE)
     gCfgItems.wifi_mode_sel = STA_MODEL;
     gCfgItems.fileSysType   = FILE_SYS_SD;
     gCfgItems.wifi_type     = ESP_WIFI;
-  #endif
+  //#endif
```

Comment:
Was it a mistake to comment out this `#if`?

Reply:
I think it's a test.. but we can remove the `#if`